crypto/dh_extra/dh_asn1.c

```diff
@@ -158,3 +158,5 @@ int i2d_DHparams(const DH *in, uint8_t **outp) {
   }
   return CBB_finish_i2d(&cbb, outp);
 }
+
+DH *DH_get_2048_256(void) { return NULL; }
```

Comment:
I don't think this will work, there code calls [uniqueptr reset](https://www.cplusplus.com/reference/memory/unique_ptr/reset/) with the value from `DH_get_2048_256` which is now null and sets it to dh2048. Which means they will pass null to `SSL_CTX_set_tmp_dh` on line 197.

Reply:
Added the RFC value function